src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class StringTest {
+    @Test
+    void _1_2을_콤마로_split_했을_때_1과_2로_분리된다() {
+        String input = "1,2";
+        String delimiter = ",";
+        assertThat(input.split(delimiter)).containsExactly("1","2");
```

Comment:
```suggestion
        assertThat("1,2".split(",")).containsExactly("1","2");
```
로컬 변수 없이 구현해도 되지 않을까?

Reply:
다른 구분자나 숫자의 테스트를 한다면? 이라는 생각으로 나눠본 것 같은데,
현재 요구 사항에서는 굳이 필요하지는 않은 것 같기도 하네요😂